.github/workflows/tests.yml

```
@@ -0,0 +1,27 @@
+name: runtests
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.16.x]
+        platform: [ubuntu-latest]
```

This change will make it easier to add additional versions and platforms in the future with minimal diffs.

```suggestion
        go-version: 
          - 1.16
        platform: 
          - ubuntu-latest
```